### PR TITLE
update to alpaka 0.8.0-dev

### PR DESCRIPTION
**Pull request by psychocoderHPC**
cupla 0.4.0 will be shipped with alpaka 0.8.0 and we skip alpaka 0.7.0 because there are too many CMake issues which are solved in 0.8.0.

```
GIT_AUTHOR_NAME="Third Party" GIT_AUTHOR_EMAIL="cupla@hzdr.de" git subtree pull --prefix alpaka git@github.com:alpaka-group/alpaka.git develop --squash
```

- remove CXX extension for clang-cuda 
- remove clang++-8 tests (linker issues)

### CODE REVIEW

**Comment by sbastrakov** [comment]
Something breaks in the CI.

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Dependencies
 ------------
 
 - **cmake 3.15.0**
```
Comment (sbastrakov) [inline README.md]: Should this requirement be updated now?

Comment (psychocoderHPC) [reply README.md]: gggrrrr yes, should be updated

Comment (psychocoderHPC) [reply README.md]: Updated

**Comment by psychocoderHPC** [comment]
ready to merge

**Review by sbastrakov — APPROVED** [review]

**Pull request merged**